tasks/sbd.yml

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: MIT
+---
+# TODO: make creating SBD headers idempotent
+# - name: Check SBD device headers
+#   command: "sbd -d {{ item }}  dump"
+#   run_once: true
+#   ignore_errors: true
+#   register: "sbd_dump_{{ item }}"
+#   loop: "{{ ha_cluster_sbd_devices }}"
+#   when:
+#     - ha_cluster_use_sbd
+#     - ha_cluster_sbd_devices
+#   tags:
+#     - sbd
+
+- name: Create SBD devices
+  command: "sbd -d {{ item }} {{ ha_cluster_sbd_create_params }} create"
+  run_once: true
+  loop: "{{ ha_cluster_sbd_devices }}"
+  when:
+    - ha_cluster_use_sbd
+    - ha_cluster_sbd_devices
+  tags:
+    - sbd
```

Comment:
What will this tag be used for?

Reply:
They were used for testing purposes - I will remove them.